.golangci-kal.yml

```diff
@@ -58,33 +58,21 @@ issues:
   max-issues-per-linter: 0
   exclude-rules:
   # KAL should only run on API folders.
-  - path-except: "api/*"
-    linters:
-      - kal
-  # apiwarnings contains a `api` characters in the path, but it's not an API folder.
-  - path: "util/apiwarnings/*"
+  - path-except: "api//*"
     linters:
       - kal
-  - path: "api/v1beta1/*|api/v1alpha1/*"
+  - path: "api//*"
     text: "Conditions field must be a slice of metav1.Condition"
     linters:
       - kal
+  # We follow the current CustomResourceDefinition field's json tag pattern.
   - path: "api/v1beta1/*"
-    text: "field XPreserveUnknownFields json tag does not match pattern"
-    linters:
-      - kal
-  - path: "api/v1beta1/*"
-    text: "field XValidations json tag does not match pattern"
-    linters:
-      - kal
-  - path: "api/v1beta1/*"
-    text: "field XMetadata json tag does not match pattern"
-    linters:
-      - kal
-  - path: "api/v1beta1/*"
-    text: "field XIntOrString json tag does not match pattern"
+    source: "X.*"
     linters:
       - kal
+  # These reporting are false positives.
+  # When the following issue is fixed, remove the exclusion.
+  # https://github.com/JoelSpeed/kal/issues/43
```

Comment:
I don't actually understand what that issue is? The PR merged to fix this no?

Reply:
Could it be that we just didn't bump KAL since the fix was merged?

Reply:
I use the [1ad4d048de47e0fa31c48e8f5d2ebe9eded0f4e4](https://github.com/JoelSpeed/kal/commit/1ad4d048de47e0fa31c48e8f5d2ebe9eded0f4e4) in https://github.com/kubernetes-sigs/cluster-api/pull/11890/commits/e48fa3cb5cbfc6f826ba1bf414f7a29bbfebb65f.
I expect this rule is removed since the extractjsontags analyzer is fixed, but this report are still be existing.
Am I missing something ?

Reply:
Ok I see, we fixed the comments linter, but not the json tags linter itself, will need a small fix there too